app/js/ui/CreateAccount.js

```diff
@@ -53,8 +53,17 @@ export default class CreateAccount extends React.Component {
 
     @autobind @preventDefault
     _handleClick() {
-        // TODO supply account info.
-        App.dispatchAction('createAccount');
+        App.dispatchAction('createAccount', {
+            form: {
+                username: this.refs.username.getDOMNode().value,
+                password: this.refs.password.getDOMNode().value,
+                favBeers: this.refs.favBeers.getDOMNode().value,
+                zipcode: this.refs.zipcode.getDOMNode().value,
+                latitude: this.refs.latitude.getDOMNode().value,
+                longitude: this.refs.longitude.getDOMNode().value,
+                favStyle: this.refs.beerStylesSelect.refs.favStyle.getDOMNode().value,
```

Comment:
TODO here that we might want to expose a getter on our custom react component so you dont have to break "Rule of Delemeter" :)